website/content/en/docs/kubebuilder/advanced-topics.md

```diff
@@ -60,14 +59,15 @@ import (
   routev1 "github.com/openshift/api/route/v1"
 )
 
-func main() {
+func init() {
   ...
 
   // Adding the routev1
-  if err := routev1.AddToScheme(mgr.GetScheme()); err != nil {
-    log.Error(err, "")
-    os.Exit(1)
-  }
+
+	utilruntime.Must(clientgoscheme.AddToScheme(mgr.GetScheme()))
+
+	utilruntime.Must(routev1.AddToScheme(mgr.GetScheme()))
+	// +kubebuilder:scaffold:scheme
```

Comment:
The indentation here is off. Use all tabs or spaces.
![image](https://user-images.githubusercontent.com/6660644/81983845-d8c59480-95e8-11ea-9074-466a99f5f3d2.png)